Makefile

```diff
@@ -0,0 +1,71 @@
+ANDROID_HOME=$(PWD)/../android
+export ANDROID_HOME
+
+BUILD_TOOLS_VERSION != sed -n 's/^ *- build-tools-//p' .travis.yml
+ANDROID_PLATFORM_VERSION != (sed -n 's/^ *- yes | sdkmanager "platforms;android-\([0-9]*\)"/\1/p' .travis.yml; echo 26) | head -n1
```

Comment:
What is the fixed `echo 26` for? Isn't this a hard-coded version number which should be fetched from `.travis.yml` as well?